### DISCUSSION TITLE
Error in =corfu--auto-complete-deferred= After Update: (wrong-type-argument arrayp nil) in Telega Completion Setup Function

### DISCUSSION THREAD

**Discussion by LuciusChen** [minad/corfu#480] [Q&A] [answered]
I have a function like this, used to start corfu for completion in telega. However, after [Use completion-metadata-get from Compat 30](https://github.com/minad/corfu/commit/daab849b05ace651df195617303e152f928980f7) update, an error occurred: `corfu--auto-complete-deferred': (wrong-type-argument arrayp nil). I don't know how to modify it. Can you help me modify this function?

```
(defun +telega-completion-setup ()
  (make-variable-buffer-local 'completion-at-point-functions)
  (setq completion-at-point-functions
        (append (mapcar #'cape-company-to-capf telega-company-backends)
                completion-at-point-functions))
  (corfu-mode 1))
```

**Comment by minad** [comment, marked as answer]
I've got a similar report here: https://github.com/minad/marginalia/issues/175#issuecomment-2218038731

This should be fixed by https://github.com/minad/corfu/commit/062e4f08d63232c31cccde5055124b8e473cf455.
